woocommerce/woocommerce-android
# Feature/make blaze dismissable

Closes: #10240 


### Description

This PR addresses the issue of the Blaze view on My Store tab not being dismissible. We've had several users complaining the view is very intrusive and not dismissible. The following changes were added to this PR: 

- Display the Blaze section below revenue stats but above the empty screen when stats are empty
- Makes Blaze view on my store tab dismissible 
- Save the state of Blaze view dismissed in shared prefs
- When a Blaze campaign is created, show Blaze module on my store tab again 
- Add new tracking event `blaze_view_dismissed` with `source:my_store_section` to track users dismissing Blaze

BONUS:
- Created a overflow menu Compose component
- Remove Blaze local feature flag

### Testing instructions

1. Open a Blaze eligible site with at least one product published
2. Check Blaze view is displayed in My Store tab
3. Dismiss Blaze view and check the following event is tracked: `blaze_view_dismissed, Properties: {"source":"my_store_section"`
4. Kill the app and re open it. Check Blaze view is gone.
5. Create a new Blaze campaign and check Blaze module is displayed again in my store tab

### Images/gif


**Hide Blaze**

https://github.com/woocommerce/woocommerce-android/assets/2663464/021027be-90cf-47b4-baf7-61bae1e80589

**Show again upon campaign created**



https://github.com/woocommerce/woocommerce-android/assets/2663464/382292fb-4368-4534-ad93-95075a0c926e
